lib/travis/build/appliances/setup_filter.rb

```diff
@@ -105,7 +105,7 @@ def exports
           end
 
           def secrets
-            @secrets ||= env_secrets.concat(data.secrets).uniq
+            @secrets ||= env_secrets.concat(data.secrets).concat(data.vault_secrets).uniq
```

Comment:
any chance data.vault_secrets will be nil? or the vault service runs always?

Reply:
> any chance data.vault_secrets will be nil? or the vault service runs always?

```
def vault_secrets
  Array(data[:vault_secrets])
end
```
```
Array(nil) => []
```